GPU/Vulkan/TextureCacheVulkan.cpp

```diff
@@ -766,11 +679,15 @@ void TextureCacheVulkan::BuildTexture(TexCacheEntry *const entry) {
 	}
 
 	int scaleFactor = standardScaleFactor_;
-	if (scaleFactor > maxScaleFactor_)
-		scaleFactor = maxScaleFactor_;
+	bool hardwareScaling = g_Config.bTexHardwareScaling && uploadCS_ != VK_NULL_HANDLE;
+	if (hardwareScaling) {
+		_assert_(shaderScaleFactor_ != 0);
```

Comment:
Hm, a misconfigured shader ini could trigger this?  Maybe better to just have it as part of the if?

-[Unknown]

Reply:
It's already checked during loading the shaders, by ignoring shaders that don't have valid scale factors. I'll just remove this assert.